android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -269,40 +270,46 @@ class RNOneginiSdk(reactContext: ReactApplicationContext) : ReactContextBaseJava
                 )
             }
         }
+    }
+
+    @ReactMethod
+    fun cancelBrowserRegistration(promise: Promise) {
+        try {
+            oneginiSDK.registrationRequestHandler.cancelRegistration()
+            return promise.resolve(null)
+        } catch (exception: OneginiReactNativeException) {
+            promise.reject(exception.errorType.toString(), exception.message)
+        }
+    }
 
+    @ReactMethod
+    fun cancelCustomRegistration(message: String?, promise: Promise) {
+        when (message) {
+            null -> promise.rejectWithNullError(Message.paramName, Message.type)
+            else -> {
+                for (action in oneginiSDK.simpleCustomRegistrationActions) {
+                    try {
+                        action.returnError(Exception(message))
+                        return promise.resolve(null)
+                    } catch (exception: OneginiReactNativeException) {}
+                }
```

Comment:
I don't like that we have a `for` here. It's not clear from outside that this method tries to cancel every possible action. I think it would be nicer if we just add idp id param and searched for it. WDYT?